scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+# Ubuntu 18.04 LTS (Bionic Beaver)
+FROM ubuntu:bionic
+
+# === INSTALL BROWSER DEPENDENCIES ===
+
+# Install WebKit dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)